draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2174,14 +2187,14 @@ uCDN-A                         dCDN-B                         dCDN-C
       <section anchor="time-policy" title="TimePolicy Extension">
         <t>uCDN may wish to perform content management operations on dCDN on a specific schedule.
            The TimePolicy extension allows uCDN to instruct dCDN to execute the trigger command
-           in a desired time window. For example, a content provider that wishes to pre-populate a new
+           in a desired time window. For example, a content provider wishes to pre-populate a new
```

Comment:
You are correct that this should be changed, but I think "may wish" would be more correct than "wishes".